HaxeManual/12-target-details.tex

```diff
@@ -108,6 +106,16 @@ \subsection{Using external JavaScript libraries}
 extern class Video {
 ..
 \end{lstlisting}
+The standard compilation options also provide more Haxe sources to be added to the project:
+
+\begin{itemize}
+	\item To add a \tref{Haxelib library}{haxelib}, use \expr{-lib <library-name>}. There are many externs for popular native libraries.
+	\item To add another class path, use \expr{-cp <directory>}.
```

Comment:
I'm fine with mentioning -lib because it is related to JS externs, but -cp seems a bit out of place here.